### PR TITLE
Refactor airflow-notebook package back into elyra core

### PR DESCRIPTION
**Pull request by akchinSTC**
- Update documentation and remove airflow package references

### How was this pull request tested?

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: etc/airflow/requirements-elyra.txt [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# This is a comprehensive list of python dependencies that Elyra requires to execute Jupyter notebooks.
```
Comment (lresende) [inline etc/airflow/requirements-elyra.txt:1]: Could we use the same for kfp and airflow? see `etc/kfp/requirements-elyra.txt`

File: etc/airflow/requirements-elyra.txt [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+nbclient==0.4.1
+nbconvert==5.6.1
+nbformat==5.0.7
+papermill==2.1.2
+pyzmq==19.0.1
+prompt-toolkit==3.0.5
+requests==2.23.0
+tornado==6.0.4
+traitlets==4.3.3
+urllib3==1.25.9
```
Comment (lresende) [inline etc/airflow/requirements-elyra.txt:17]:
```suggestion
urllib3==1.26.5
```

File: etc/airflow/requirements-elyra.txt [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# This is a comprehensive list of python dependencies that Elyra requires to execute Jupyter notebooks.
+ipykernel==5.3.0
+ipython==7.15.0
+ipython-genutils==0.2.0
+jupyter-client==6.1.6
+jupyter-core==4.6.3
+minio==5.0.10
```
Comment (lresende) [inline etc/airflow/requirements-elyra.txt:7]:
```suggestion
minio==6.0.2
```

File: etc/airflow/requirements-elyra.txt [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+nbclient==0.4.1
+nbconvert==5.6.1
+nbformat==5.0.7
+papermill==2.1.2
+pyzmq==19.0.1
+prompt-toolkit==3.0.5
+requests==2.23.0
```
Comment (lresende) [inline etc/airflow/requirements-elyra.txt:14]:
```suggestion
requests==2.25.1
```

File: etc/airflow/requirements-elyra.txt (hunk 2 above)
Comment (lresende) [reply etc/airflow/requirements-elyra.txt:17]: There is a security fix on urllib3 which requires the version changes above

File: etc/airflow/requirements-elyra.txt (hunk 1 above)
Comment (akchinSTC) [reply etc/airflow/requirements-elyra.txt:1]: yes we probably can, lets move the requirements up a level or define a new agnostic directory for it

**Review by lresende — APPROVED** [review]

**Pull request merged**